open-grant-proposals/open-proposal-filecoin-spec-localization.md

```diff
@@ -0,0 +1,69 @@
+# Open Grant Proposal : `Filecoin Spec Localization`
+
+**Name of Project :**  Filecoin Spec Localization
+
+**Proposal Category :**  docs
+
+**Proposer :** [@maobisheng](github.com/maobisheng)
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses? :** Yes 
+
+# Project Description
+
+Our team feels that it is necessary to create a Chinese version of the Filecoin Spec. In addition, we also plan to generate 
+multilingual versions of `filecoin-project-specs`, including but not limited to Chinese and English.
+## Value
+
+We find that Chinese miners and researchers account for the majority of the filecoin market, and most of them cannot 
+fully understand the `Filecoin Specification` due to their inadequate English proficiency. Chinese developers are often distressed by some 
+simple theoretical knowledge, which is shown in slack and other communities. We always see Chinese users asking 
+questions that can be easily found in the Spec. 
+
+## Deliverables
+
+- Adding a language toggle button to the Spec page
+- Adding language packs for the source code of `filecoin-project-specs`
+
+## Budget
+
+All for basic cost : $1000
+
+## Maintenance and Upgrade Plans
+
+We would focus on the Filecoin Spec updates in real time and make changes accordingly. 
+
+# Team
+
+## Team Members
+
+- BiSheng Mao : Proposer   
+- Huan Shen : Interpreter
+- WenYi Li : Interpreter
+- XueFeng Bu : Filecoin Engineer and Researcher
+ 
+## Team Member Github Profiles
+
+- https://github.com/maobisheng (personal)
+- https://github.com/XQYCHJ (personal)
+- https://github.com/Amorous13 (personal)
+
+## Team Website
+
+  [@IPFSUNION](https://ipfsunion.cn)
+
+## Relevant Experience
+
+Our team has been focusing on Filecoin's mining testing, specification studies, problem submission, and code 
+contributions. 
+
+Our developers have been reading block chain English documents for years and paying close attention to the source
+code, specification and implementation of Filecoin research. 
+
+## Team Translation Example
+
+The translation of Section 2.3 for Filecoin Specification:
+[link of jianshu](https://www.jianshu.com/p/544a2c07366d)
```

Comment:
Hi @maobisheng

I'm getting a 404 for https://www.jianshu.com/p/544a2c07366d do you have an updated link for this translation?

Reply:
Yes, please find below links:
https://juejin.im/post/5e12ec1cf265da5d6f68c83d
https://juejin.im/post/5e12edf4f265da5d363e308d
https://juejin.im/post/5e12ee1ce51d4541375f1389
https://juejin.im/post/5e1300c3e51d454126369bb5